### DISCUSSION TITLE
Welcome to Time-Appliance-Project Discussions!

### DISCUSSION THREAD

**Discussion by ahmadexp** [Time-Appliances-Project/Time-Card#85] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rudivandrunen** [comment]
Hi all,

Just was wondering if the time stick was commecially available. I was planning to buy 2 ea to experiment with ptp.

Rudi
